rubygem-smart_proxy_discovery_image/rubygem-smart_proxy_discovery_image.spec

```diff
@@ -0,0 +1,86 @@
+%global gem_name smart_proxy_discovery_image
+
+%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+Name: rubygem-%{gem_name}
+Version: 1.0.3
+Release: 1%{?dist}
+Summary: Discovery image (node) plugin for Foreman's smart proxy
+Group: Applications/Internet
+License: GPLv3
+URL: http://github.com/theforeman/smart_proxy_discovery_image
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: ruby(rubygems)
+Requires: foreman-proxy >= 1.7.0
+Requires: rubygem(rest-client) > 1.6.2
+Requires: rubygem(rest-client) < 1.7
+
+
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+BuildRequires: ruby(abi)
+%else
+Requires: ruby(release)
+BuildRequires: ruby(release)
+%endif
+BuildRequires: rubygems-devel
+BuildRequires: ruby(rubygems)
+
+BuildArch: noarch
+
+Provides: rubygem(%{gem_name}) = %{version}
+Provides: foreman-proxy-plugin-discovery-image
+
+%description
+This smart proxy plugin is not intended for direct use. It is installed
+on foreman discovery image which is running on discovered nodes.
```

Comment:
Capitalise Foreman Discovery?  It also doesn't match the gemspec, it's probably worth syncing them.